azure-pipelines.yml

```diff
@@ -144,6 +144,7 @@ stages:
             pythonVersion: ${{ parameters.minimumPythonVersion }}
             testQPY: false
             testImages: false
+            rustVersion: "1.56.1"
```

Comment:
Personally I'd define this as a "parameter" called `minimumSupportedRustVersion` even within this file - this file doesn't actually take any inputs, but it lets us attach a name to the magic number.  We do that for `minimumPythonVersion`, etc, for example.

I don't feel wildly strongly about it, though.